Expensify/App
# Create a desktop layout for the home page

### Details
This adds a desktop layout to the login form

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/148719

### Tests
1. Open the app
2. Be signed out
3. Verify the log in form looks good on all platforms

### Screenshots

![image](https://user-images.githubusercontent.com/1228807/102940291-366a6b00-446d-11eb-9aed-545a10a09e7d.png)

![image](https://user-images.githubusercontent.com/1228807/102940313-41250000-446d-11eb-9974-0449e95db282.png)
